airbyte-integrations/connectors/source-gocardless/source_gocardless/manifest.yaml

```diff
@@ -49,7 +49,6 @@ streams:
   - type: DeclarativeStream
     $parameters:
       name: "payments"
-      stream_cursor_field: "created_at"
```

Comment:
Some cases where the stream_cursor_field was defined but unneeded by a full refresh only streams